### ISSUE TITLE
Blood Fury (Orc racial) scaling improperly

### ISSUE THREAD

**Issue by Opilol** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#133]
The orc racial, Blood Fury, is currently not working properly and hasn't for awhile. It scales overall attack power at the current state of use rather than base attack power.

**Comment by devidao** [comment]
how can i reproduce this?

**Comment by Ifixstuffk** [comment]
Make a character, test how much AP Bloodfury adds naked. 
Add a ton of Strength and test it again, calculate difference

**Comment by sycvil** [comment]
The question is what should be considered as base attack power. Some people support the idea above which supposes that base AP = AP of a completely naked and unbuffed char. I thought that base AP = white AP (160 on the picture),
![attack power](https://cloud.githubusercontent.com/assets/11522469/18014842/935c413a-6bcd-11e6-8931-2b02976b2851.JPG)
although I can't find a source for any of those ideas.

Why the first assumption (base AP = AP of a naked char) seems to be not quite correct to me. There is a somewhat similar spell [Enrage](http://www.wow-one.com/database/index.php?spell=5229&realm=0) which reduces base Armor by a certain percent. If we suppose that base AP = AP of a naked unbuffed char, it would be logical to think that base Armor = Armor of a naked unbuffed char as well (a level 60 naked Druid has ~110 armor). But then Enrage would only remove ~18 armor on a geared level 60 Druid in Dire Bear Form which is kind of a joke. And Enrage doesn't work like that, it removes much more Armor on geared chars and considers armor from items. This makes me think that base AP/Armor is more than just AP/Armor of a naked unbuffed char.

**Comment by devidao** [comment]
I agree with sycvil, firstly need describe what is base attack power.
